src/fontra/client/core/path-functions.js

```diff
@@ -114,3 +114,24 @@ export function connectContours(path, sourcePointIndex, targetPointIndex) {
   }
   return new Set([`point/${selectedPointIndex}`]);
 }
+
+export function deleteSelectedPoints(path, pointIndices) {
+  for (const pointIndex of reversed(pointIndices)) {
+    const [contourIndex, contourPointIndex] = path.getContourAndPointIndex(pointIndex);
+    const numContourPoints = path.getNumPointsOfContour(contourIndex);
+
+    if (numContourPoints > 1) {
+      path.deletePoint(contourIndex, contourPointIndex);
+    } else {
+      path.deleteContour(contourIndex);
+    }
+  }
+}
+
+export function getSelectedContours(path, pointIndices) {
```

Comment:
Since you didn't end up using this function in deleteSelectedPoints, I would prefer it to stay where it was. I'm not yet convinced it is a super generally useful function.

Reply:
This PR should effectively leave scene-controller.js alone.

Reply:
No problem, will revert back to the old version.